README.md

```diff
@@ -14,8 +14,10 @@ go get -u github.com/NYTimes/openapi2proto/cmd/openapi2proto
 There are 4 CLI flags for using the tool:
 * `-spec` to point to the appropriate OpenAPI spec file
 * `-annotate` to include (google.api.http options) for [grpc-gateway](https://github.com/gengo/grpc-gateway) users. This is disabled by default.
-* `-out` to have the output written to a file rather than `Stdout. Defaults to `Stdout` if this is not specified`
+* `-out` to have the output written to a file rather than `Stdout`. Defaults to `Stdout` if this is not specified.
 * `-indent` to override the default indentation for Protobuf specs of 4 spaces.
+* `-generateRpcs=false` to disable generation of rpcs. This is enabled by default.
+* `-prefixEnums` to enable inserting the enum name as an enum prefix for each value. This is disabled by default.
```

Comment:
Ditto for the camel case here, maybe `-prefix-enums`? I also wonder if we can come up with a better name